spec/googleauth/compute_engine_spec.rb

```diff
@@ -152,16 +152,46 @@ def make_auth_stubs opts
     end
   end
 
-  context "metadata is unavailable" do
+  context "metadata is available" do
     describe "#fetch_access_token" do
-      it "should pass scopes when requesting an access token" do
+      it "should pass scopes" do
         scopes = ["https://www.googleapis.com/auth/drive", "https://www.googleapis.com/auth/bigtable.data"]
         stub = make_auth_stubs access_token: "1/abcdef1234567890", scope: scopes
         @client = GCECredentials.new(scope: scopes)
         @client.fetch_access_token!
         expect(stub).to have_been_requested
       end
+    end
+
+    describe "Fetch ID tokens" do
```

Comment:
why is this a separate describe with the same name as below?

Reply:
Different context (metadata is available vs metadata is not available)